src/screens/GroupInfo/elements/Media.js

```diff
@@ -4,6 +4,7 @@ import {FlatList} from 'react-native';
 import {StyleSheet, View} from 'react-native';
 import {Context} from '../../../context';
 import useMedia from './useMedia';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the `utils/theme` module is a good change. It ensures that color values are consistent across the application and can be easily updated in one place.
```diff
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->